utils/compress.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python3
+from datetime import datetime
+import os
+import sys
+sys.path.append("..")
+import settings
+import re
+import tarfile
+import settings
+import boto3
+import logging
+
+S3_BUCKET_NAME = getattr(settings, "S3_BUCKET_NAME", None)
+S3_ACCESS_KEY_ID = getattr(settings, "S3_ACCESS_KEY_ID", None)
+S3_SECRET_ACCESS_KEY = getattr(settings, "S3_SECRET_ACCESS_KEY", None)
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+class Compress:
+    def __init__(self, extension):
+        self.ext = extension
+        self.secret = ""
+        self.output = None
+        self.output_fn = datetime.now().isoformat()
+        self.open()
+
+    def open(self):
```

Comment:
It doesn't seem like `open` is intended as a public interface.
Let's change it to `_open` to communicate that. Still should have a docstring.